### DISCUSSION TITLE
함께 자라기 책 수요 조사

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#80] [General]
저를 비롯해 몇몇 분은 책이 있는데

책이 없으신 분은 책을 선물해 드리도록 하겠습니다.

책이 없어서 저에게 받으실 분은 댓글을 달아 주시면 e-book으로 전달 드리겠습니다.

**Comment by momoci99** [comment]
저요!!!

**Reply by jongfeel** [reply]
드렸습니다!
![image](https://user-images.githubusercontent.com/17442457/147450673-5d04e3e2-f9f5-4f03-92c2-f2bc0e5dc689.jpeg)

**Comment by Joseph-Cha** [comment]
저요!!!

**Reply by jongfeel** [reply]
드렸습니다!
![image](https://user-images.githubusercontent.com/17442457/147455505-eae03410-bc4d-4d0c-b850-23b6a220144b.png)

**Comment by hi-hyein** [comment]
저요오오~~~!

**Reply by jongfeel** [reply]
드렸습니다!
![image](https://user-images.githubusercontent.com/17442457/147519162-36d49ab3-9aae-4171-9706-50eceefdc39c.jpeg)

**Comment by jongfeel** [comment]
생각해 보니 Academic conference 2022라고 적었어야 했는데, 아직 2022년이 되지 않아서 그랬나 봅니다 ㅎㅎ
